src/Oakton/CommandLineHostingExtensions.cs

```diff
@@ -23,7 +23,7 @@ public static class CommandLineHostingExtensions
         /// <returns></returns>
         public static Task<int> RunOaktonCommands(this IHostBuilder builder, string[] args)
         {
-            return Execute(builder, null, args);
+            return Execute(builder, Assembly.GetEntryAssembly(), args);
```

Comment:
@jeremydmiller not sure how you feel about this change. I didn't see how else to get commands that are declared in the application assembly to be added when using the `RunOaktonCommands` extension.

Reply:
That's a good oversight to get fixed, yeah. Used to depend on walking up the call stack, but that doesn't play nicely in every circumstance so I ditched it. I agree with that, but I'll need to change the docs a bit.

Reply:
Hmm... thinking about it a bit more. What if my commands are not defined in the web application entry assembly? Seems like there needs to be some overload or optional parameters on `RunOaktonCommands` to add individual assemblies or tell it to scan all assemblies. But maybe that is what `OaktonCommandAssemblyAttribute` is for, eh?

Reply:
That's what `OaktonCommandAssemblyAttribute` is for:) Sorry for the delay on this one, I'm getting in and released with Oakton 3.1 today.